dashboard/run_data_collection.py

```diff
@@ -28,6 +29,34 @@
 PROJECT_ID = "nesi00213"
 TIME_FORMAT = "%Y_%m_%d-%H_%M_%S"
 HPCS = [hpc.value for hpc in const.HPC]
+USERS = [
+    "ykh22",
+    "cbs51",
+    "melody.zhu",
+    "tdn27",
+    "jpa198",
+    "leer",
+    "baes",
+    "sjn87",
+    "tdn27",
+    "jmotha",
+    "jagdish.vyas",
+    "ddempsey",
+]
+USERS_REAL_NAME = {
+    "ykh22": "Jonney Huang",
+    "cbs51": "Claudio Schill",
+    "melody.zhu": "Melody Zhu",
+    "tdn27": "Andrei Nguyen",
+    "jpa198": "James Paterson",
+    "leer": "Robin Lee",
+    "baes": "Sung Bae",
+    "sjn87": "Sarah Neill",
+    "jmotha": "Jason Motha",
+    "jagdish.vyas": "jagdish Vyas",
+    "ddempsey": "David Dempsey",
+}
+UTC_TIME_GAP = datetime.now() - datetime.utcnow()
```

Comment:
Maybe make this a class variable, since its used directly by the DataCollector class